todo_app/tests_e2e/test_integration_mongodb_repository.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from todo_app.data.mongodb_repository import mongodb_repository
+import random
+import string
+import logging
+import pymongo
+
+from todo_app.data.task import TaskStatus
+
+connection_string = "mongodb+srv://jmaw1:ppppp@cluster0.5vzof.mongodb.net/doMeDatabase?retryWrites=true&w=majority"
```

Comment:
This is your genuine username and password - we should definitely remove this and change your MongoDb password!

Reply:
Done thanks!